features/support/paths.rb

```diff
@@ -262,6 +262,12 @@ def path_to(page_name)
     when /^the [mM]y account page$/
       '/my/account'
 
+    when /^the [aA]ccess [tT]oken page$/
+      '/my/access_token'
+
+    when /^the [sS]ettings page$/
```

Comment:
Hint: in the backlogs plugin there is a cuke using this step:

> When I go to the my settings page

Not sure where this step originates...